.github/workflows/publish-docker-dev-image.yml

```diff
@@ -19,5 +19,5 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
           registry: docker.pkg.github.com
           repository: remotecloud/kplex/kplex
```

Comment:
After merge you need to rename this line to:  `repository: stripydog/kplex/kplex` . I don't rename it since I use my fork to get the latest docker image (till this is merged).